Dockerfile

```diff
@@ -37,13 +37,13 @@ RUN composer install --no-dev && \
 
 # Build Content
 COPY bin/build-content ./bin/build-content
-RUN apk add --no-cache ruby ruby-json ruby-bigdecimal ruby-dev build-base libxml2-dev libxslt-dev libffi-dev && \
+RUN apk add --no-cache ruby ruby-json ruby-bigdecimal ruby-dev build-base libxml2-dev libxslt-dev libffi-dev yaml-dev && \
```

Comment:
Hey, it looks good to me, before approving it, do you mind explaining why we need this `yaml-dev` library please?

Reply:
bump

Reply:
to install `rdoc` which is included in a previous step in the Dockerfile, it needs this `yaml-dev` library to install correctly. If we don't include this library, this error pops up in the installation:

![image](https://user-images.githubusercontent.com/65265704/224326402-c242cf23-55bc-45c2-a6a7-aea5b1f3b845.png)

this didn't happen before because with the `alpine:edge` image, we were sitting on version `3.15.0_alpha20210804` instead of version `3.15.0`, which might not have that necessary `yaml-dev` library.

Reply:
thanks!